railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -284,6 +287,14 @@ def initialize(*args)
           self.options = options.merge(skip_sprockets: true, skip_javascript: true).freeze
         end
 
+        # Skip yarn for minimal apps, actionmailer, actioncable, webpacker, activestorage
+        if options[:minimal]
+          self.options = options.merge(skip_webpack_install: true, skip_javascript: true,
+            skip_active_storage: true, skip_action_cable: true, skip_action_mailer: true,
+            skip_turbolinks: true, skip_spring: true, skip_action_text: true, skip_bootsnap: true,
+            skip_action_mailbox: true).freeze
```

Comment:
If we arrange these options one-per-line and in alphabetical order, it would make this list easier to examine and update in the future.